### PR TITLE
Updating RT onboarding doc with a link to prow commands

### PR DESCRIPTION
**Pull request by savitharaghunathan**
Updating RT onboarding doc with a link to prow commands

#### What type of PR is this:

/kind documentation

#### What this PR does / why we need it:
Adding prow commands link to the RT onboarding doc will come in handy and also let new contributors familiarize themselves with the automation.

cc: @justaugustus

### CODE REVIEW

File: release-team/release-team-onboarding.md [hunk 1]
```diff
@@ -40,6 +40,10 @@ All members of the current release team should be apart of this list. If you're
 
 All members of the release team and anyone who attends Release Team meetings / keeps up with the release process should join the SIG Release group: https://groups.google.com/forum/#!forum/kubernetes-sig-release
 
+### Familiarize with Prow commands
+
+Kubernetes uses Prow to manage CI automation. Issues and PRs are categorized by applying suitable labels, which helps in prioritization of that specific issue/PR during the release cycle. Learn more about the available labels and prow commands from here: https://prow.k8s.io/command-help?repo=kubernetes%2Fkubernetes
```
Comment (justaugustus) [inline release-team/release-team-onboarding.md:45]:
```suggestion
Kubernetes uses [Prow](https://git.k8s.io/test-infra/prow/README.md) to manage CI automation. Issues and PRs are categorized by applying suitable labels, which helps in prioritization of that specific issue/PR during the release cycle. Learn more about the available labels and prow commands from here: https://prow.k8s.io/command-help
```

**Review by justaugustus — CHANGES_REQUESTED** [review]
Thanks @savitharaghunathan!

Two nits:
- Describe/link to Prow
- Drop the repo query from the command help link

I've made both as a code suggestion.

File: release-team/release-team-onboarding.md (hunk 1 above)
Comment (savitharaghunathan) [reply release-team/release-team-onboarding.md]: Fixed it :) Thanks for the review, @justaugustus

**Comment by justaugustus** [comment]
thanks again for adding this!
/lgtm
/approve

**Pull request merged**